script/bulk_import/base.rb

```diff
@@ -579,8 +580,11 @@ def pre_cook(raw)
     cooked = @markdown.render(cooked).scrub.strip
 
     cooked.gsub!(/\[QUOTE="?([^,"]+)(?:, post:(\d+), topic:(\d+))?"?\](.+?)\[\/QUOTE\]/im) do
-      username, post_id, topic_id = $1, $2, $3
-      quote = @markdown.render($4.presence || "").scrub.strip
+      username, post_id, topic_id, quote = $1, $2, $3, $4
+
+      quote = quote.scrub.strip
```

Comment:
Why did you remove the markdown rendering? Sometimes, a quote has markdown/html and we need to render it.